apps/user-portal/src/components/account-recovery-email.tsx

```diff
@@ -0,0 +1,220 @@
+    const {onNotificationFired} = props;
+
+    const handleUpdate = () => {
+        const data = {
+            Operations: [
+                {
+                    op: "replace",
+                    value: {}
+                }
+            ],
+            schemas: [
+                "urn:ietf:params:scim:api:messages:2.0:PatchOp"
+            ]
+        };
+
+        data.Operations[0].value = {
+            emails: [editedEmail]
+        };
+
+        updateProfileInfo(data)
+            .then((response) => {
+                if (response.status === 200) {
+                    onNotificationFired({
+                        description: t(
+                            "views:accountRecovery.emailRecovery.notification.success.description"
+                        ),
+                        message: t(
+                            "views:accountRecovery.emailRecovery.notification.success.message"
+                        ),
+                        otherProps: {
+                            positive: true
+                        },
+                        visible: true
+                    });
+                    getProfileInfo()
+                        .then((res) => {
+                            setEmailAddr(res);
+                        });
+                    setIsEdit(false);
+                } else {
+                    onNotificationFired({
+                        description: t(
+                            "views:userProfile.notification.getProfileInfo.error.description"
+                        ),
+                        message: t(
+                            "views:userProfile.notification.getProfileInfo.error.message"
+                        ),
+                        otherProps: {
+                            negative: true
+                        },
+                        visible: true
+                    });
+                }
+            });
+    };
+
+    /**
+     * The following function handles the change of state of the input fields.
+     * The id of the event target will be used to set the state.
+     * @param event
+     */
+    const handleFieldChange = (event) => {
+        setEditedEmail(event.target.value);
+        event.preventDefault();
+    };
+
+    const setEmailAddr = (resp) => {
+        let emailAddress = "";
+        resp.emails.map((email) => {
+            emailAddress = email;
+        });
+        setEmail(emailAddress);
+        setEditedEmail(emailAddress);
+    };
+
+    const handleEdit = () => {
+        setIsEdit(true);
+    };
+
+    const handleCancel = () => {
+        setIsEdit(false);
+    };
+
+    const showEditView = () => {
+        if (!isEdit) {
+            return (<Grid padded>
+                    <Grid.Row columns={2}>
+                        <Grid.Column width={11} className="first-column">
+                            <List.Content floated="left">
+                                <ThemeIcon
+                                    icon={MFAIcons.sms}
```

Comment:
Let's have a separate icon for email recovery

Reply:
This has been rectified. 
![image](https://user-images.githubusercontent.com/20745428/66899943-7d630200-f019-11e9-82e6-859a60966de1.png)